src/phonemizer.cpp

```diff
@@ -1,6 +1,43 @@
 #include "phonemizer.h"
 
-/*
+#ifdef ESPEAK_INSTALL
+/**
+ * espeak_wrapper functions
+ * 
+ * The espeak_wrapper is a singleton which wraps threaded calls to espeak-ng with a shared mutex
+ */
+espeak_wrapper * espeak_wrapper::get_instance() {
+	if (!instance) {
+		instance = new espeak_wrapper;
+	}
```

Comment:
ggml backend registry uses the simpler solution of just putting a static variable here.